src/eradiate_plugins/bsdfs/rtls.cpp

```diff
@@ -0,0 +1,322 @@
+#include <mitsuba/core/frame.h>
+#include <mitsuba/core/fwd.h>
+#include <mitsuba/core/math.h>
+#include <mitsuba/core/properties.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/core/warp.h>
+#include <mitsuba/render/bsdf.h>
+#include <mitsuba/render/texture.h>
+
+NAMESPACE_BEGIN(mitsuba)
+
+/**!
+
+.. _plugin-bsdf-rtls:
+
+Ross-Thick Li-Sparse reflection model (:monosp:`rtls`)
+--------------------------------------------------------
+
+.. pluginparameters::
+
+ * - f_iso
+   - |spectrum| or |texture|
+   - :math:`f_{iso}`. Default: 0.209741
+   - |exposed| |differentiable|
+
+ * - f_geo
+   - |spectrum| or |texture|
+   - :math:`f_{geo}`. Default: 0.081384
+   - |exposed| |differentiable|
+
+ * - f_vol
+   - |spectrum| or |texture|
+   - :math:`f_{vol}`. Default: 0.004140
+   - |exposed| |differentiable|
```

Comment:
RTLS does not really treat h, r and b as parameters for the model, because inversion tools usually fix these values beforehand. But for our implementation, We may treat all the parameters the same way. I will correct this.